flash/graph/classification/data.py

```diff
@@ -94,3 +98,11 @@ def num_features(self):
         n_cls_val = getattr(self.val_dataset, "num_features", None)
         n_cls_test = getattr(self.test_dataset, "num_features", None)
         return n_cls_train or n_cls_val or n_cls_test
+
+    from_folders = None
```

Comment:
@ethanwharris Do we want to do this for now? Just an idea, but might not be worth it.